bwmenu

```diff
@@ -285,10 +326,10 @@ show_copy_notification() {
   local body=""
   local extra_options=()
 
-  title="<b>$(echo "$1" | jq -r '.name')</b> copied"
+  title="$(echo "$1" | jq -r '.[0].name') copied"
```

Comment:
The point of *show_copy_notification* is that only 1 item is passed. I think the other functions should make sure to pass only the object, not an array of objects.

Reply:
In this instance we already made sure the list has only one item, but yeah, it's cleaner if we pass the item itself.